TESTING-INSTRUCTIONS.md

```diff
@@ -269,10 +282,19 @@ localStorage.setItem( 'debug', 'wc-admin:tracks' );
         -   The enable/disable button should be correctly reflected in the Woocommerce payment settings screen as well.
 
 Once everything is configured and enabled do a client test
+<<<<<<< HEAD
```

Comment:
It appears your auto formatter got to this file, and it somehow managed to get some merge conflicts in

Reply:
Whoops thanks.

Reply:
Fixed.